.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,9 +13,11 @@ about: Create a report to help us improve
 [troubleshooting]: https://github.com/autozimu/LanguageClient-neovim/blob/next/INSTALL.md#6-troubleshooting
```

Comment:
The linked section needs to be updated with this PR.

Reply:
Great catch! I'll run a check for other references to any of the markdown files.